README.md

```diff
@@ -1,7 +1,7 @@
-Barefoot Nomad - Making company travel and accomodation easy and convinient.
-=======
+[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
+
+# Barefoot Nomad - Making company travel and accomodation easy and convinient.
```

Comment:
Just replace the word "convinient" with convenient

Reply:
Thanks @WillyBoris for the grammatical correction!

Reply:
> Just replace the word "convinient" with convenient otherwise It looks good to merge

Thanks @WillyBoris for the grammatical correction!